src/core/manager.c

```diff
@@ -50,6 +50,7 @@
 #include "io-util.h"
 #include "label.h"
 #include "locale-setup.h"
+#include "load-fragment.h"
```

Comment:
@poettering would say that the order is wrong, but when I squint, `o` and `a` look the same ;)